packages/forge2d/lib/src/dynamics/world.dart

```diff
@@ -22,7 +22,27 @@ class World {
   final List<Joint> joints = <Joint>[];
 
   final Vector2 _gravity;
+
+  /// The current [World]'s [gravity].
+  ///
+  /// {@template World.gravity}
+  /// All [bodies] are affected by [gravity]; unless the [Body] has a specified
+  /// [Body.gravityOverride]. If so, [Body.gravityOverride] is used instead of
+  /// [gravity].
+  ///
+  /// See also:
+  ///
+  /// * [Body.gravityScale], to multipy [gravity] for a [Body].
+  /// * [Body.gravityOverride], to change how the world treats the gravity for
+  /// a [Body].
+  /// {@endtemplate}
   Vector2 get gravity => _gravity;
+
+  /// Changes the [World]'s gravity.
+  ///
+  /// {@macro World.gravity}
+  set gravity(Vector2 gravity) => this.gravity.setFrom(gravity);
```

Comment:
I think we can remove the getter and the setter completely (previously `setGravity` and getter `gravity`). Similar to #63.

Since one can already get the `gravity` from the getter and due to vectors being mutable alter it outside the setter.

Wdyt @spydon ?

Reply:
Do you think we can provide documentation to solve this issue?

I was thinking a video series for Forge2D and a section under the Flame website. What do you think?

Also I think the current setter encourages new users to create a new vector. Which is more expensive that providing the values.

Reply:
Sure, we can consider it solved by good docs. I think that some users will still be confused though (people are bad at reading docs), but just like you say we don't want them to create too many new objects when it can be avoided.

It would be good if this PR gets a follow-up PR that adds that to the flame_forge2d docs at least.

Reply:
Sounds like a plan to me. 

I can work on those docs, however it will take a decent amount of time since I want to ensure high quality.

I would like to include demo code or video snippets that showcase how gravity, gravityScale and gravityOverride work in an illustrated way.

Reply:
Sounds good, I'll discuss with the rest of blue fire whether we want videos in the docs or not though. But we have the possibility to have snippets with running flutter apps embedded in the docs already, like here:
https://docs.flame-engine.org/main/flame/inputs/tap-events.html#demo